tests/Doctrine/Tests/Models/Cache/Token.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Doctrine\Tests\Models\Cache;
+
+/**
+ * @Entity
+ * @Cache("READ_ONLY")
```

Comment:
It should work with any cache mode. Not having the associations resolve definitely sounds like a bug, no matter the cache mode :)